cmake/multi_image.cmake

```diff
@@ -78,7 +78,7 @@ else()
     file(
       APPEND
       ${base_image_preload_file}
-      "set(${app_var_name} ${${app_var_name}} CACHE INTERNAL \"NCS child image controlled\")\n"
+      "set(${app_var_name} \"${${app_var_name}}\" CACHE INTERNAL \"NCS child image controlled\")\n"
```

Comment:
you should also update this line:
https://github.com/nrfconnect/sdk-nrf/blob/277e9ce74207be14dc99fd03825e6cb332c9e5ba/cmake/multi_image.cmake#L380